sdk-core-protos/src/lib.rs

```diff
@@ -30,9 +30,9 @@ pub mod coresdk {
         fmt::{Display, Formatter},
         iter::FromIterator,
     };
-    use workflow_activation::{wf_activation_job, WfActivationJob};
+    use workflow_activation::{workflow_activation_job, WorkflowActivationJob};
     use workflow_commands::{workflow_command, workflow_command::Variant, WorkflowCommand};
-    use workflow_completion::{wf_activation_completion, WfActivationCompletion};
+    use workflow_completion::{workflow_activation_completion, WorkflowActivationCompletion};
 
     #[allow(clippy::module_inception)]
     pub mod activity_task {
```

Comment:
For the most part I don't think it's best to add hand-written code to the same module as the generated code for extension purposes.

https://github.com/temporalio/sdk-core/blob/2fe8665cbcf0048c91440589a6276728bbbc8113/sdk-core-protos/src/lib.rs#L1148-L1153

That is a reasonable proto package name and mimics a name I'd like to do for the core protos (not the `.v1` suffix since internal, but `temporal.sdk.core` package prefix). But I have not yet because of the concern around this ugliness. Thoughts?

Reply:
The implementations can be moved to different modules in the same crate if needed, though I'm not really sure it buys us much since all the actual generated code is still in separate files.

The package renaming I'd only bother with if it really makes a real improvement in another lang or something. I get that it'd look more cannonical but I'm not sure we care much given this isn't really a public API